Hack-A-Sat 2021/Presents From Marco/Groundead/README.md

```diff
@@ -0,0 +1,154 @@
+# Hack-a-Sat "Presents From Marco" / "Grounded"
```

Comment:
Can you please fill out the top section the way that we have done other Hack-a-Sat writeups? See Hack-A-Sat 2021/Rapid Unplanned Disassembly/Grade F Prime Beef for an example.